source/managing_account/02_managing_api_keys/managing_api_keys.rst

```diff
@@ -0,0 +1,53 @@
+Managing API keys
+=================
+
+After authenticating to a portal, you can generate API keys and manage them from the API keys tab of their account.
```

Comment:
The phrase is written with "you" but switched to "their" at the end (instead of "your" I suppose).